SasquatchTV/SasquatchTVSwift/AppDelegate.swift

```diff
@@ -16,7 +16,8 @@ class AppDelegate : UIResponder, UIApplicationDelegate, MSCrashesDelegate {
 
     // Override point for customization after application launch.
     MSAppCenter.setLogLevel(MSLogLevel.verbose);
-    MSAppCenter.start("fbdeeff6-a549-4653-9f6d-f97dfe7f07b4", withServices : [MSAnalytics.self, MSCrashes.self]);
+    MSAppCenter.setLogUrl("https://in-integration.dev.avalanch.es")
+    MSAppCenter.start("e57f6975-9167-4b3b-b450-bbb87b717b82", withServices : [MSAnalytics.self, MSCrashes.self]);
```

Comment:
This is OK only as a temporary solution

Reply:
What is the name of that app and why does it work only on INT?

Reply:
Why is it a temporary solution. Is production not ready? Same for the other app.

Reply:
This tvOS swift gets a 401 from ingestion, the app secret is probably wrong.